src/visuals.js

```diff
@@ -1180,7 +1211,7 @@ export const Visuals = {
     // }
   },
 
-  drawBody(x, y, v, radius, body) {
+  drawBody(x, y, v, radius, body, backgroundOnly = false) {
```

Comment:
recommend just throwing `backgroundOnly` on `body` so that you don't have to thread it through to drawBaddie

Reply:
updated!